demos/demo-scatter-plot.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const d3Selection = require('d3-selection');
+const PubSub = require('pubsub-js');
+
+const scatterPlot = require('./../src/charts/scatter-plot');
+const colors = require('./../src/charts/helpers/color');
+const dataBuilder = require('./../test/fixtures/scatterPlotDataBuilder');
+
+const aTestDataSet = () => new dataBuilder.ScatterPlotDataBuilder();
+
+require('./helpers/resizeHelper');
+
+function createSimpleScatterPlot() {
+        let scatter = scatterPlot();
+        let scatterPlotContainer = d3Selection.select('.js-simple-scatter-plot-container');
+        let containerWidth = scatterPlotContainer.node() ? scatterPlotContainer.node().getBoundingClientRect().width : false;
+        let dataset;
+
+        if (containerWidth) {
+            dataset = aTestDataSet().withFourNames().build();
+
+            scatter
+                .width(750)
+                .maxCircleArea(15);
+
+            scatterPlotContainer.datum(dataset).call(scatter);
+        }
+}
```

Comment:
Should we ask Sun for a loading state for this chart? I don't think we have something even remotely close to it...

Reply:
@Golodhros that would be a great idea! Once it's ready we can easily add it to the chart.